pkgs/sdk_triage_bot/lib/triage.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:github/github.dart';
+
+import 'src/common.dart';
+import 'src/gemini.dart';
+import 'src/github.dart';
+import 'src/prompts.dart';
+
+final sdkSlug = RepositorySlug('dart-lang', 'sdk');
+
+Future<void> triage(
+  int issueNumber, {
+  bool dryRun = false,
+  bool force = false,
+  required GithubService githubService,
+  required GeminiService geminiService,
+}) async {
+  print('Triaging $sdkSlug...');
```

Comment:
As we move to productionize this and use it consistently in the long term this we'll  probably want to take out the `print` in favor of a more log driven approach. I don't think we should block on that for now.

Reply:
👍